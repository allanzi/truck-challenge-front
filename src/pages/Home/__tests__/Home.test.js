import React from 'react';

import HomePage from '../index';

it('should render correct', () => {
  const wrapper = shallow(<HomePage />);

  expect(wrapper).toMatchSnapshot();
});
