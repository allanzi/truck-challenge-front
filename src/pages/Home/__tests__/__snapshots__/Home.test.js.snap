// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correct 1`] = `
<Fragment>
  <Breadcrumb
    separator="/"
    style={
      Object {
        "margin": "16px 0",
      }
    }
  >
    <BreadcrumbItem
      separator="/"
    >
      Truck Panel
    </BreadcrumbItem>
    <BreadcrumbItem
      separator="/"
    >
      Home
    </BreadcrumbItem>
  </Breadcrumb>
  <div
    className="site-layout-background"
    style={
      Object {
        "minHeight": 360,
        "padding": 24,
      }
    }
  >
    <Row
      gutter={
        Array [
          8,
          8,
        ]
      }
    >
      <Col
        span={8}
      >
        <Card
          className="card-report"
          title="Viagens por caminhoneiro"
        >
          <Timeline
            mode=""
            reverse={false}
          />
        </Card>
      </Col>
      <Col
        span={8}
      >
        <Card
          className="card-report"
          title="Caminhoneiros carregados"
        >
          <List
            bordered={true}
            dataSource={Array []}
            loading={true}
            pagination={false}
            renderItem={[Function]}
            split={true}
          />
        </Card>
      </Col>
      <Col
        span={8}
      >
        <Card
          className="card-report"
          title="Caminhoneiros com veículo próprio "
        >
          <List
            bordered={true}
            dataSource={Array []}
            loading={true}
            pagination={false}
            renderItem={[Function]}
            split={true}
          />
        </Card>
      </Col>
    </Row>
    <Row
      gutter={
        Array [
          8,
          8,
        ]
      }
    >
      <Col
        span={8}
      >
        <Card
          className="card-report"
          title="Qtde caminhões carregados por dia"
        >
          <Timeline
            mode=""
            reverse={false}
          />
        </Card>
      </Col>
      <Col
        span={8}
      >
        <Card
          className="card-report"
          title="Qtde caminhões carregados por semana"
        >
          <Timeline
            mode=""
            reverse={false}
          />
        </Card>
      </Col>
      <Col
        span={8}
      >
        <Card
          className="card-report"
          title="Qtde caminhões carregados por mês"
        >
          <Timeline
            mode=""
            reverse={false}
          />
        </Card>
      </Col>
    </Row>
  </div>
</Fragment>
`;
